.github/ISSUE_TEMPLATE.md

```diff
@@ -29,9 +29,20 @@ modinfo spl | grep -iw version
 ### Describe how to reproduce the problem
 
 ### Include any warning/errors/backtraces from the system logs
+
 <!-- 
-Example:
+
+Hints for formatting
+
 ```
-this is an example how log text should be marked (wrap it with ```)
+wrap commands with ```
 ```
+
+    prefix multiline output with four spaces. an easy way is to pipe it to sed 's/^/\ \ \ \ /g'
```

Comment:
the three backticks method didn't seem to work reliably for me while i found that the four spaces always produced monospaced text.
sorry i'm not too much into github markup formatting.
the idea is to make it more visibile to the user that monospaced text that comes from a terminal should always be formatted in such a way that it would get displayed monospaced on github.
the current template does not insist too much on this aspect.